### PR TITLE
Build/docker dev env

### PR DESCRIPTION
**Pull request by anoff**
This should bring a new docker-compose file that runs the whole ecosystem.. maybe? :D

### CODE REVIEW

File: setup/docker-compose.yml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+    - ../lib:/app/lib
+    - ../bin:/app/bin
+    - ../data:/app/data
+    - ../mine.js:/app/mine.js
+    - ../config.js:/app/config.js
+    entrypoint: "./node_modules/.bin/nodemon bin/cli.js train --mine-address 0xE8cd631a35daEA6c76cF9a8c04C1a326fE69f9A9 --contract-address 0xdde11dad6a87e03818aea3fde7b790b644353ccc --ethereum-url http://testrpc:8545"
+    networks: [mine-dev]
+    depends_on: [testrpc]
+  notebook:
+    image: openmined/pysyft-notebook:edge
```
Comment (kevinahuber) [inline setup/docker-compose.yml:26]: Is this supposed to be `pysyft:edge`? if not, can you please push this image to dockerhub 😈

Comment (anoff) [reply setup/docker-compose.yml:26]: Waiting for my PR on the psySyft repo to go through. Then I'll set up automated builds for the Dockerfile.notebook.

**Review by iamtrask — APPROVED** [review]
Woohoo!

File: setup/docker-compose.yml (hunk 1 above)
Comment (anoff) [reply setup/docker-compose.yml:26]: Awww shoot you're somehow right it's supposed to be `pysonar-notebook`

**Pull request merged**